### ISSUE TITLE
Missing `principal_committe_id` for Candidates with multiple `candidate_ids` 

### ISSUE THREAD

**Issue by andrewthornton** [fecgov/openFEC#3872]
I recently downloaded the Candidate Master data in bulk from [here](https://www.fec.gov/files/bulk-downloads/2020/cn20.zip). In processing the data, I noticed that for a few candidates that have run multiple times and have multiple `candidate_id`s, they are missing the `principal_committee_id` for their most recent run. Two candidates I found that exhibited this were `SIEGEL, MIKE` and `KEAN, THOMAS H. JR.` They have a `principal_committee_id` for their previous campaigns, but none for their current one (2020). Any ideas why this data might be missing?

**Comment by PaulClark2** [comment]
@andrewthornton thank you for using our data offerings and contacting the Federal Election Commission. For both Siegel and Kean a duplicate candidate ID was created in error. Our reports processing group will correct this by expunging the newer ID. They will then correct any candidate committee linkages.  

- Siegel: H0TX10208 is duplicate of H8TX10110.
- Kean: H0NJ07261 is duplicate of H0NJ07089 

There are a few scenarios  where a candidate ID will end up in the CM file without a PCC. 
- The candidate files a Statement of Candidacy (FEC Form 2) and never files a Statement of Organization (FEC Form 1) for his or her PCC. 
- Some candidates are entered into our database from state ballot lists. If a candidate is entered into our database from a state ballot list, the candidate may not file a Form 1 for his or her PCC. Please note, the ballot list candidate might not be required to file a Form 1 depending on the campaigns level of financial activity.
- Sometimes it is just timing. There can be as much as a 10-day gap between filing a Form 2 and a Form 1. 

If you find other issues in our data, please let us know.

**Comment by andrewthornton** [comment]
@PaulClark2, thank you for your response. I also noticed that Tulsi Gabbard has the same PCC for her presidential run as her house run. Her candidate id for each run is different, however. Is this an error?

**Comment by PaulClark2** [comment]
@andrewthornton yes, it's a mistake. Ms. Gabbard's links for 2020 should look like this: 

- H2HI02508 link to C00497396
- P00009183 link to C00693713.

I've removed the incorrect link between her House candidate ID and her presidential PCC. You should see that correction in the file tomorrow.

**Comment by andrewthornton** [comment]
On a recent data pull, it appears that several candidates have stale committee ids in the bulk data. The candidates below all have two committees. However, in the bulk data the committee id appears to be for the inactive committee

**CO-03: Diane Mitsch Bush**
C00707638 - active
C00649459

**FL-27: Maria Elvira Salazar**
C00714261 - active
C00671859

**MI-03: Jon Hoadley**
C00701599 - active
C00701581

**MO-03: Blaine Leutkemeyer**
C00458679 - active
C00447748
